tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/PoPTests.NetFwk.cs

```diff
@@ -58,7 +58,7 @@ public void TestInitialize()
 
             if (_keyVault == null)
             {
-                _keyVault = new KeyVaultSecretsProvider(KeyVaultInstance.MsalTeam);
+                _keyVault = new KeyVaultSecretsProvider(KeyVaultInstance.MSIDLab);
```

Comment:
there are only two more secret references after this that points to `KeyVaultInstance.MsalTeam`, please file a work item for us to move those secrets to `msidlabs` vault and remove `KeyVaultInstance.MsalTeam` references

Reply:
No there are 4. 2 for OBO 2 in Selenium tests